Frontend/Program.cs

```diff
@@ -19,10 +19,12 @@ public static void Main(string[] args)
         }
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
-            Host.CreateDefaultBuilder(args)
-                .ConfigureWebHostDefaults(webBuilder =>
-                {
-                    webBuilder.UseStartup<Startup>();
-                });
+                    Host.CreateDefaultBuilder(args)
+                        .ConfigureWebHostDefaults(webBuilder =>
+                        {
+                            webBuilder
+                            .UseSetting(WebHostDefaults.DetailedErrorsKey, "true")
```

Comment:
Good that you added this. 😄 Now we can see full error/exception messages in chrome console.